### DISCUSSION TITLE
Braille glyphs not showing on terminal

### DISCUSSION THREAD

**Discussion by Cassiano-Gunji** [aristocratos/btop#250] [Q&A] [answered]
I'm running a Raspberry Pi with Open Media Vault, which turns a Raspbian Lite install into a NAS. When running btop on a SSH connection (PuTTY), it runs beautifully, but when I run it on the terminal (HDMI monitor connected to the Raspberry Pi), only TTY characters are shown.

I've played a lot with console-terminal configurations. It behaves as expected, except I can't make the terminal to show braille characters.

Is it possible to display braille characters on the terminal? If so, how to do it?

![20220117_233652](https://user-images.githubusercontent.com/23004311/149861481-d45ad06d-c72c-4c87-988d-51c26c07259a.jpg)

![image](https://user-images.githubusercontent.com/23004311/149861512-2513dcb9-1a3b-4a07-a3ef-5271b2e688a3.png)

**Comment by aristocratos** [comment, marked as answer]
@Cassiano-Gunji 
> Is it possible to display braille characters on the terminal? If so, how to do it?

It probably possible with some custom framebuffer driver and custom fonts, but not in a standard linux console with standard fonts.
Btop automatically detects if running in a real tty so you also need to start with `--tty_off` to disable tty mode in the console.

**Reply by Cassiano-Gunji** [reply]
That's about what I feared. Also, starting with --tty_off gets even uglier. It seems I'll just have to accept to live with those TTY symbols. Thank you!
